### ISSUE TITLE
logbackextension: Throwable from error log is missing in LogEntry 

### ISSUE THREAD

**Issue by Daniel-80** [logrecorder/logrecorder#30]
When using the logbackextension the throwables von log.error(message, throwable) are missing in the logrecorder-LogEntries

In our Company we have tests where we want to assure, that those errors are logged to the appender, which we actually cant do with Version 2.3.0 of the logrecorder

It would be great, if you could extend the LogRecord with this information.

The logback appender for example creates the logmessage bei getting formattedmessage and throwable:

```
public String doLayout(ILoggingEvent event) {
        if (!isStarted()) {
            return CoreConstants.EMPTY_STRING;
        }
        StringBuilder sb = new StringBuilder();
        long timestamp = event.getTimeStamp();
        sb.append(cachingDateFormatter.format(timestamp));
        sb.append(" [");
        sb.append(event.getThreadName());
        sb.append("] ");
        sb.append(event.getLevel().toString());
        sb.append(" ");
        sb.append(event.getLoggerName());
        sb.append(" - ");
        sb.append(event.getFormattedMessage());
        sb.append(CoreConstants.LINE_SEPARATOR);
        IThrowableProxy tp = event.getThrowableProxy();
        if (tp != null) {
            String stackTrace = tpc.convert(event);
            sb.append(stackTrace);
        }
        return sb.toString();
    }
```

It  would be great, if you could extend the LogRecord with this message.

**Comment by slu-it** [comment]
Hi @Daniel-80 thank you for your suggestion.

We are looking into providing the original throwable as part of the `LogEntry`.
This has been on our Radar for some time, but we never got around to it.

Providing the original `Throwable` should be a more flexible way for actually asserting desired outcomes than having the formatted message, that might change over time.

As soon as this is available I'll update this issue.

**Comment by slu-it** [comment]
@Drakojin added support for Throwables as well as some additional features and changes in #31 

I just triggered the release of v2.4.0 (https://github.com/logrecorder/logrecorder/releases/tag/v2.4.0) - please take note of the breaking change described in that release.

**Comment by Daniel-80** [comment]
Hi @slu-it ,
thank you for your fast reaction and implementation!

In our compony we get the artefacts from maven- central (https://search.maven.org/artifact/io.github.logrecorder/logrecorder). I do not find the new Version 2.4.0 there. Is that an oversight or intentional?

**Comment by Daniel-80** [comment]
Hi @slu-it ,
the artifact is now available in maven-central in Version 2.4.0. The test of the new version was successful (i.e. we can now also check the throwable) 
Thanks again for the quick implementation!
